docs/config.rst

```diff
@@ -0,0 +1,35 @@
+Configuration
+=============
+
+rST Blog expects a YAML file with some configuration values. For example::
```

Comment:
Not sure how far we are going with docs for now, but this leads to the follow-up questions: what are the valid configuration settings? What do they mean?